web/vtadmin/src/api/http.test.ts

```diff
@@ -17,7 +17,6 @@ import { rest } from 'msw';
 import { setupServer } from 'msw/node';
 
 import * as api from './http';
-import { vtadmin as pb } from '../proto/vtadmin';
```

Comment:
Was this removed just because it was unused?

Reply:
Yeah, it actually added a nice annotation + everything:

<img width="1678" alt="Screen Shot 2021-03-08 at 9 22 39 PM" src="https://user-images.githubusercontent.com/855595/110408932-6e3c8400-8054-11eb-8212-d5b271b29668.png">

Reply:
love to see it, sincerely